ui/components/MesheryPerformanceComponent.js

```diff
@@ -134,6 +135,7 @@ class MesheryPerformanceComponent extends React.Component {
   };
 
   handleSubmit = () => {
+    const pattern = new RegExp('^https?:\\/\\/' + '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|' + '((\\d{1,3}\\.){3}\\d{1,3}))' + '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*' + '(\\?[;&a-z\\d%_.~+=-]*)?' + '(\\#[-a-z\\d_]*)?$', 'i');
     const {
```

Comment:
Can we avoid defining pattern at the start of the handleSubmit function, as it may confuse future contributors?

Reply:
So should I declare it globally?

Reply:
@rshivam08 , by avoiding in the start @dhruv0000 meant that declare it just before the if condition where we are checking the pattern, correct me if I am wrong.

Reply:
Yup.